video.c

```diff
@@ -702,11 +705,14 @@ video_advance_crtc_timing(struct video_struct* p_video) {
         }
       }
 
+      int is_teletext = (p_video->video_ula_control & k_ula_teletext);
```

Comment:
I tried to adhere to strict C, i.e. only declarations at the head of a block. Can you move this line to just after the "uint16_t address_counter = p_video->address_counter;" line?

Reply:
Yes, done.